packages/holocron-dev-server/src/middleware/parrot-scenarios.js

```diff
@@ -90,7 +90,7 @@ export async function createHotParrotMiddleware({
         const mockFiles = watched[mockPath];
         return mockFiles.map((fileName) => path.join(mockPath, fileName));
       })
-      .reduce((array, next) => array.concat(next), []);
+      .reduce((array, next) => [...array, next], []);
```

Comment:
this is not the same. Array.prototype.concat allows for next to be an array or not and wont result in nested arrays. 
`[].concat('something') === [].concat(['something'])` 
`[...array, next] !== [...array, [next]]`